lib/index.d.ts

```diff
@@ -0,0 +1,250 @@
+import { AWS } from '@serverless/typescript';
+
+declare module '@serverless/typescript' {
+  interface AWS {
+    stepFunctions?: {
+      stateMachines: StateMachines;
+      validate?: boolean;
+    };
+  }
+}
+
+type StateMachines = {
+  [stateMachine: string]: StateMachine
+};
+
+export type StateMachine = {
+  type?: 'STANDARD' | 'EXPRESS';
+  id?: string;
+  name?: string;
+  definition: Definition;
+  tracingConfig?: TracingConfig;
```

Comment:
```suggestion
  tracingConfig?: TracingConfig;
  loggingConfig?: LoggingConfig;
  events?: any[];
  dependsOn?: string | string[];
};

type LoggingConfig = {
  level: "ERROR" | "ALL" | "FATAL" | "OFF";
  includeExecutionData: boolean;
  destinations: Resource | Resource[]
```
We forgot that one in one subcomment.